script.js

```diff
@@ -3,4 +3,11 @@ var tempString = prompt("What is your temperature?");
 var tempNum = Number(tempString)
 
 
-// Put your if logic here
\ No newline at end of file
+// Put your if logic here
+if (tempNum <= 98.6) {
+  alert("You do not have a fever");
+}
+
+if (tempNum > 98.6) {
```

Comment:
When this program runs it will check both of these conditions regardless of whether or not the first one is satisfied. In order to prevent this, use `else if (tempNum > 98.6)`